src/inmanta/protocol/methods_v2.py

```diff
@@ -670,6 +670,37 @@ def resource_logs(
     """
 
 
+@typedmethod(
+    path="/resource/<rid>/facts", operation="GET", arg_options=methods.ENV_OPTS, client_types=[ClientType.api], api_version=2
+)
+def get_facts(tid: uuid.UUID, rid: model.ResourceIdStr) -> List[model.Parameter]:
+    """
+    Get the facts related to a specific resource
+    :param tid: The id of the environment
+    :param rid: Id of the resource
+    :return: The facts related to this resource
+    :raise NotFound: This exception is raised when the referenced environment is not found
```

Comment:
In the API documentation, the exception name is automatically translated to the corresponding status code. So the description should not talk about an exception.
```suggestion
    :raise NotFound: This status code is returned when the referenced environment is not found
```